test/screens/home/home_view_model_test.dart

```diff
@@ -128,5 +133,20 @@ void main() {
         mockGetSurveysUseCase.call(any),
       ]);
     });
+
+    test('When call refresh, it call getCachedSurveysUseCase', () async {
```

Comment:
Please be more careful with grammar issues 🙏 
```suggestion
    test('When calling refresh method, it calls getCachedSurveysUseCase', () async {
```